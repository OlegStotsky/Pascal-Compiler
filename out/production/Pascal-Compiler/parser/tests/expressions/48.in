var
	r1 : record
		r2 : record
			x : float;
		end;
	end;


var
	y : integer;

begin
	y := r1.r2.x;
end;