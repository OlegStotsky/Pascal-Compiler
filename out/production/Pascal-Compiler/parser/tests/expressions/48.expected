Error at line 12, column 15 : can't cast type float to type integer
